cvat-ui/src/components/annotation-page/top-bar/player-navigation.tsx

```diff
@@ -39,6 +39,7 @@ interface Props {
     onSliderChange(value: number): void;
     onInputChange(value: number): void;
     onURLIconClick(): void;
+    onCopyFrameFileNameIconClick(): void;
```

Comment:
Name looks too complex

```suggestion
    onCopyFilenameIconClick(): void;
```